app/src/main/java/com/novoda/downloadmanager/demo/DemoApplication.java

```diff
@@ -26,24 +20,10 @@ public void onCreate() {
     }
 
     private void createLiteDownloadManager() {
-        FileSizeRequester fileSizeRequester = new CustomFileSizeRequester();
-        FileDownloader fileDownloader = new CustomFileDownloader();
-        DownloadsPersistence downloadsPersistence = new CustomDownloadsPersistence();
-        NotificationCreator notificationCreator = new CustomNotificationCreator(this, R.mipmap.ic_launcher_round);
-
         Handler handler = new Handler(Looper.getMainLooper());
 
         liteDownloadManagerCommands = DownloadManagerBuilder
                 .newInstance(this, handler, R.mipmap.ic_launcher_round)
-                .withFileDownloaderCustom(fileSizeRequester, fileDownloader)
```

Comment:
There's a whole bunch of custom classes here to demonstrate the uses of the library but for the moment we just want to work with the vanilla version 😄